pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -135,6 +137,8 @@
   curriculum_name_csf_02: "Any CS Fundamentals (Course A-F)"
   curriculum_name_csd: "Computer Science Discoveries"
   curriculum_name_csp: "Computer Science Principles"
+  curriculum_name_csc: "Computer Science Connections"
```

Comment:
This string isn't used anywhere (yet), but I added it for consistency w/ the other courses.